GeneratorInterface/GenFilters/src/PythiaMomDauFilter.cc

```diff
@@ -0,0 +1,161 @@
+
```

Comment:
We tried to use PythiaDauFilter and PythiaFilter but in our use case we need the link between mother-daugther-nephew that combining the two filters separetely doesn't work.